Resources/Prototypes/_RMC14/Entities/Clothing/Head/Helmets/marine_helmets.yml

```diff
@@ -105,6 +105,34 @@
       Classic: _RMC14/Objects/Clothing/Head/Helmets/m10/medic/classic.rsi
       Urban: _RMC14/Objects/Clothing/Head/Helmets/m10/medic/urban.rsi
 
+- type: entity
+  parent: RMCMarineHelmetBase
```

Comment:
You should be able to parent this off the other corpsman helmet so that you only need to copy the sprite part in its components, not the whole thing